src/contentChunk.js

```diff
@@ -61,3 +61,29 @@ exports.extractStringPayloadIntoChunks = (payload) => {
     stream.end(payload)
   })
 }
+
+export const retrievePayload = (fileId, callback) => {
+  const db = connectionDefault.client.db()
+  if (!db) {
```

Comment:
We dont need this logic to check the db. The connection is made in the connection.js file and only used here. Maybe what we can do is to create a common function for getting the GridFS bucket which can be used by both functions.

```
const getGridFSBucket = () => {
   return new mongodb.GridFSBucket(connectionDefault.client.db())
}
...
const bucket = getGridFSBucket()
```